vistas/contacto.php

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="es">
+    <head>
+        <?php include("../logica/header.php"); ?>
+        <title>Contacto Back Music</title>
+    </head>
+    <body>
+        <div id="contenedor">
+            <?php include("../logica/cabecera.php"); ?>
+            <?php include("../logica/lateralIzq.php"); ?>
+            <main>
+                <article>
+                    <section>
+                        <h2>Por favor, rellene el siguiente formulario para contactar con nosotros:</h2>
+                        <form action="mailto:contactoBackMusic@ucm.es" method="post" enctype="text/plain"> <!-- correo inventado -->
```

Comment:
Si queréis mantener el formulario de contacto, hay que implementarlo de verdad con PHP. Podéis utilizar la funcion mail() de PHP para enviar emails y verificar su contenido / evío en mail.containers.fdi.ucm.es